frontend/src/pages/dashboard/mock-dashboard-data.js

```diff
@@ -0,0 +1,49 @@
+const sampleFeed = [
+    {
+        id: 1,
+        content: 'Post 1',
+        author: {
+            id: 1,
+            username: 'poster1',
+            nickname: 'Poster 1',
+            profilePic: 'https://i.imgur.com/qZImY9j.jpg',
+        },
+        parent: null,
+        children: [5],
+        usersLiked: 1,
+        usersShared: 2,
+        timestamp: 1646627469012,
+    },
```

Comment:
Have you made a data contract with backend (or is this already made based on their model)?

Reply:
I gave that data in the issue which is based on the backend model, the data format will be changing slightly with the feed endpoint but I will fix that when I hook up posts properly to the backend